src/com/mendix/widget/badge/components/BadgeLabel.ts

```diff
@@ -0,0 +1,21 @@
+import { DOM, createElement } from "react";
+import * as classNames from "classnames";
+
+import { BadgeProps } from "./Badge";
+
+export const BadgeLabel = (props: BadgeProps) =>
+    createElement("div",
+        {
+            className: classNames("widget-badge-display",
+                { "widget-badge-link": !!props.onClick }
+            ),
+            disabled: props.disabled,
```

Comment:
I added it for consistency 
When a Badge Button is disabled, It appears pale and cannot be clicked. This happens when there is no context